packages/arb-token-bridge-ui/src/components/common/AddCustomChain.tsx

```diff
@@ -0,0 +1,186 @@
+import { useState } from 'react'
+import { XMarkIcon } from '@heroicons/react/24/outline'
+import { Chain, L2Network, addCustomChain } from '@arbitrum/sdk'
+
+import { Button } from './Button'
+import { ChainWithRpcUrl, getNetworkName } from '../../util/networks'
+
+export const localStorageKey = 'arbitrum-custom-chains'
+
+const jsonPlaceholder = `{ chainID: 0, name: 'MyChain', partnerChainID: 0, ...other_properties }`
+
+// allow only Ethereum testnets and Arbitrum testnets as parent chains
+const allowedParentChainIds = [5, 11155111, 1337, 421613, 421614, 412346]
```

Comment:
Hardcoded because `ChainId` is in `networks.ts` and `networks.ts` also imports stuff from this file. I don't want to move `ChainId` because of extra diff.

Lmk if you have any ideas.